src/ruleLoader.ts

```diff
@@ -57,26 +57,40 @@ export function findRule(name: string, rulesDirectory?: string) {
         return Rule;
     }
 
-    // then check for rules within the first level of rulesDirectory
-    if (rulesDirectory) {
-        Rule = loadRule(rulesDirectory, camelizedName);
-        if (Rule) {
-            return Rule;
+    // normalize the rulesDirectories argument
```

Comment:
Kept the normalization for backwards compatibility, but by using `Lint.Linter.getRulesDirectories` instead of duplicating the logic. Done in commit https://github.com/palantir/tslint/commit/ee60a0cd1b858bd9555f21e9b4efdb501ebb570d.